README.md

```diff
@@ -0,0 +1,347 @@
+## Rules Overview
+
+### Abstract class must be prefixed with Abstract
+
+- class: [`Hostnet\Sniffs\Classes\AbstractClassMustBePrefixedWithAbstractSniff`](/src/Hostnet/Sniffs/Classes/AbstractClassMustBePrefixedWithAbstractSniff.php)
+- sniff: Hostnet.Classes.AbstractClassMustBePrefixedWithAbstract
+
+:x:
+
+```php
+class MyClass
+{
+    public abstract function getEntityManager();
+}
+```
+
+:+1:
+
+```php
+abstract class MyClass
```

Comment:
We actually have a different sniff that obligates prefixing abstract classes with `Abstract`, so I would call good `AbstractMyClass`.